assets/css/frontend.scss

```diff
@@ -690,7 +690,7 @@ section.entry span.course-lesson-progress { margin-left: 10px; }
 
 /* Course Results */
 .course-results-lessons {
-  h2 {
+  h2, h3, h4 {
```

Comment:
This SCSS can be cleaned up to reduce bloat:

- If we're no longer using an `h2`, that selector can be removed.
- Since the `total-grade` element is always inside an `h3`, the CSS for it doesn't need to be applied to the `h4` as well.

The resulting SCSS would be something like this:
```
/* Course Results */
.course-results-lessons {
  h3, h4 {
    margin: 20px 0;

    .lesson-grade {
      float: right;
    }
  }

  h3.total-grade {
    text-decoration: underline;

    .lesson-grade {
      text-decoration: underline;
    }
  }
}
```

Reply:
The h2 is still applied to the "Lessons" heading, so perhaps the margin should stay for that?
`  h2 {
    margin: 20px 0;
  }`